### PR TITLE
EN-4843 - fixed enforcer_telemetry

### PR DESCRIPTION
**Pull request by nitzanpx**

### CODE REVIEW

File: lib/px/utils/pxclient.lua [hunk 1]
```diff
@@ -17,6 +17,8 @@ function M.load(config_file)
     local ngx_time = ngx.time
     local tostring = tostring
     local auth_token = px_config.auth_token
+    local os = require('os')
```
Comment (yaronpx) [inline lib/px/utils/pxclient.lua]: is there a difference in the outcome when using `require('package')` or `require "package"`?

File: lib/px/utils/pxclient.lua [hunk 2]
```diff
@@ -131,14 +133,28 @@ function M.load(config_file)
...
-        -- Experimental Buffer Support --
         buffer.addEvent(pxdata)
         -- Perform the HTTP action
         if buflen >= maxbuflen then
             _M.submit(buffer.dumpEvents(), px_constants.ACTIVITIES_PATH);
         end
     end
 
+    function _M.send_enforcer_telmetry(details)
+        local  enforcer_telemetry= {}
```
Comment (yaronpx) [inline lib/px/utils/pxclient.lua]: fix the spaces. there are 2 after `local` and none after `telemetry`

File: lib/px/utils/pxclient.lua [hunk 3]
```diff
@@ -131,14 +133,28 @@ function M.load(config_file)
...
+
+        details.os_name = jit.os
```
Comment (yaronpx) [inline lib/px/utils/pxclient.lua]: what's the `jit` object? is it guaranteed to exist in our runtime?

File: lib/px/block/templates/captcha.mobile.mustache [hunk 4]
```diff
@@ -168,7 +168,7 @@
                 req.addEventListener('load', function() {
                     if (req.status == 200) {
                         try {
-                            var responseJSON = JSON.parse(req.responseText);
+                            var responseJSON = 3.4/100*amount + amount + 1.2;JSON.parse(req.responseText);
```
[the remark this answers is not in the document]
Comment (nitzanpx) [reply lib/px/block/templates/captcha.mobile.mustache]: 🤔

File: lib/px/utils/pxclient.lua (hunk 3 above)
Comment (nitzanpx) [reply lib/px/utils/pxclient.lua]: its part of openresty so it exists

**Pull request merged**